SnapWar/Managers/SWConfigurationManager.swift

```diff
@@ -0,0 +1,26 @@
+import UIKit
+
+class SWConfigurationManager {
+    enum Environment: String {
+        case Dev = "dev"
+        case Production = "production"
+    }
+    
+
+    lazy var environment: Environment = {
+        if let configuration = Bundle.main.object(forInfoDictionaryKey: "Configuration") as? String {
+            if configuration.range(of: "Production") != nil {
+                return Environment.Production
+            }
+        }
+
+        return Environment.Dev
+    }()
+    
+    var baseApiUrl: String {
+        switch self.environment {
+            case .Dev: return "192.168.0.218:8080"
```

Comment:
@misspia change this to your local wifi address

Reply:
lol are we changing this everytime the other person changes it o-o

Reply:
i'll figure out a better way to store this somewhere. in the meantime, just avoid checking this change in to the repo 😄